docs/conf.py

```diff
@@ -56,8 +56,14 @@
     'readthedocs': ('https://docs.readthedocs.io/en/stable/', None),
     'sphinx': ('https://www.sphinx-doc.org/en/master/', None),
 }
-hoverxref_intersphinx = True
-hoverxref_intersphinx_type = 'modal'
+hoverxref_intersphinx = [
```

Comment:
Should this be `hoverxref_intersphinx_projects`? The name currently doesn't really explain what the data type inside would be. 

It seems that we're overloading it to be "enable and list of projects" -- perhaps that still works with `projects` in the name, since having `[]` projects effectively means disabled.

Reply:
> It seems that we're overloading it to be "enable and list of projects"

Yes.

> Should this be `hoverxref_intersphinx_projects`? The name currently doesn't really explain what the data type inside would be.

Hrm, yeah. I'm not super happy with the name either. However, the most correct technical name would be `hoverxref_intersphinx_mappings` since it refers to the keys from the `intersphinx_mappings` config --but I didn't like it 😞